website/docs/getting-started/setup/installation-guides/aws-ecs-on-fargate.md

```diff
@@ -18,41 +18,68 @@ This document guides you through installing Appsmith using AWS ECS on Fargate.
     Ensure that the EFS, ECS cluster, and Fargate instances are all in the same VPC.
     :::
       
-    3. Click on the EFS created and navigate to the Network tab to ensure mount-target is made in the same availability zone as the ECS cluster.
+    3. Click the EFS created and navigate to the Network tab to ensure the mount-target is made in the same availability zone as the ECS cluster.
     4. Create a new security group to allow inbound and outbound NFS traffic.
     5. Attach the security group to the EFS mount-target.
 
-5. **Create the ECS task execution role:**
+5. **Create the ECS task role:**
```

Comment:
Let's keep it as Sumesh suggested 'task execution role' . Just read up that is some difference between both

Reply:
I've changed this. @sum35h - Please confirm.

Reply:
Oh sorry, my bad. I might have been confused because in some parts of the doc, I saw task execution role.

Reply:
This is a change to what was earlit task execution role. Now it's just task role